packages/trader/src/Modules/Trading/Components/Form/WheelPicker/wheel-picker.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState, useEffect } from 'react';
+import { Button, Icon, Text } from '@deriv/components';
+import { motion } from 'framer-motion';
+
+const variants = {
+    enter: (direction: 'up' | 'down') => ({
+        x: 0,
+        y: direction === 'down' ? 30 : -30,
+        opacity: 0,
+        transition: {
+            duration: 0.24,
+            ease: [0, 0, 0, 1],
+        },
+    }),
+    center: {
+        x: 0,
+        y: 0,
+        opacity: 1,
+        transition: {
+            duration: 0.24,
+            ease: [0, 0, 0, 1],
+        },
+    },
+    exit: (direction: 'up' | 'down') => ({
+        x: 0,
+        y: direction === 'down' ? -30 : 30,
```

Comment:
Some of these styles repeat across all of them. You may move them out into a default object.